slick.grid.js

```diff
@@ -3979,8 +3977,6 @@ if (typeof Slick === "undefined") {
     // Public API
 
     $.extend(this, {
-      "slickGridVersion": "3.5",

```

Comment:
Found another place where the version number is kept - this is technically a breaking change but err... so not major version bump material.